src/devices/bus/hp80_io/82937.cpp

```diff
@@ -0,0 +1,354 @@
+// license:BSD-3-Clause
+// copyright-holders: F. Ulivi
+/*********************************************************************
+
+    82937.cpp
+
+    82937 module (HPIB interface)
+
+    TODO: Implement Parallel Poll response
+
+    Thanks to Tim Nye & Everett Kaser for dumping the 8049 ROM
+
+    Main reference for this module is:
+    HP 82937-90007, oct 80, HP82937A HP-IB Installation and theory
+    of operation manual
+
+*********************************************************************/
+
+#include "emu.h"
+#include "82937.h"
+#include "coreutil.h"
+
+// Debugging
+#define VERBOSE 0
+#include "logmacro.h"
+
+// Bit manipulation
+namespace {
+	static constexpr unsigned BIT_MASK(unsigned n)
```

Comment:
There's no need to use `static` here:
* `constexpr` implies `inline`, which suppresses symbol export already
* It's in an anonymous namespace which suppresses symbol export